client/src/buttons.ts

```diff
@@ -1,6 +1,15 @@
 import { ExtensionContext } from 'vscode';
 import * as vscode from 'vscode';
 
+const REACH_COMPILE_S:string = 'Reach Compile';
```

Comment:
nit:  Can you change the suffix of all the button text constants to use `_TEXT` instead of `_S`? 
e.g. `REACH_COMPILE_S => REACH_COMPILE_TEXT`.